Components/Alert/alert.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
not mandatory but @smile_gupta suggested this in review session
#1 you can add meta tag here ex. <meta name"author" content="your name"/>